### PR TITLE
feat: introduce Connector SDK `0.3.0`

### PR DESCRIPTION
**Pull request by tmetzke**
## What is the purpose of the change

Collect all changes necessary for Connector SDK `0.3.0`.

## Are there related marketing activities

n/a

## When should this change go live?

After the SDK `0.3.0` has been published. Scheduled for November 15th, 2022. We'll ping here manually again.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [X] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
> Scheduled for November 15th, 2022. We'll ping here manually again.
🙏 Thank you!

**Comment by tmetzke** [comment]
@christinaausley, this is ready from engineering. Happy to receive a tech writing review. 🙂

**Comment by tmetzke** [comment]
SDK `0.3.0` is released, ready to merge from our end

File: docs/guides/update-guide/connectors/introduction.md [hunk 1]
```diff
@@ -9,9 +9,14 @@ Connector runtimes to a newer version of the
 
 There is a dedicated update guide for each version:
 
+### [Connector SDK 0.2 to 0.3](../020-to-030)
+
+Update from 0.2.x to 0.3.0
+
+[//]: # "TODO: As soon as the release will be created[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.2.0)"
```
Comment (tmetzke) [inline docs/guides/update-guide/connectors/introduction.md:16]:
```suggestion
[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.3.0)"
```

File: versioned_docs/version-8.0/guides/update-guide/connectors/introduction.md [hunk 2]
```diff
@@ -9,9 +9,14 @@ Connector runtimes to a newer version of the
 
 There is a dedicated update guide for each version:
 
+### [Connector SDK 0.2 to 0.3](../020-to-030)
+
+Update from 0.2.x to 0.3.0
+
+[//]: # "TODO: As soon as the release will be created[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.2.0)"
```
Comment (tmetzke) [inline versioned_docs/version-8.0/guides/update-guide/connectors/introduction.md:16]:
```suggestion
[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.3.0)
```

File: versioned_docs/version-8.1/guides/update-guide/connectors/introduction.md [hunk 3]
```diff
@@ -9,9 +9,14 @@ Connector runtimes to a newer version of the
 
 There is a dedicated update guide for each version:
 
+### [Connector SDK 0.2 to 0.3](../020-to-030)
+
+Update from 0.2.x to 0.3.0
+
+[//]: # "TODO: As soon as the release will be created[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.2.0)"
```
Comment (tmetzke) [inline versioned_docs/version-8.1/guides/update-guide/connectors/introduction.md:16]:
```suggestion
[Release notes](https://github.com/camunda/connector-sdk/releases/tag/0.3.0)
```

**Comment by akeller** [comment]
@christinaausley are you available to review this today?

**Comment by tmetzke** [comment]
> @christinaausley are you available to review this today?

If not, we could also go for a merge and release and a follow-up PR if there are any adjustments from a tech writing perspective, I guess?

**Comment by akeller** [comment]
@tmetzke correct! I would do a quick skim review of the PR and then merge. So, either way, it will go out today.

**Comment by akeller** [comment]
@tmetzke it's on @christinaausley's list to review today.

**Review by christinaausley — APPROVED** [review]
Minor adjustments in latest commit -- let me know if you have any questions 👍 LGTM.

**Comment by tmetzke** [comment]
Feel free to merge whenever ready (I will probably be offline already by then)

**Comment by akeller** [comment]
@christinaausley there is a build issue on your commit. Can you take a look?

File: versioned_docs/version-8.1/self-managed/connectors-deployment/connectors-configuration.md [hunk 4]
```diff
@@ -102,7 +102,7 @@ to inject multiple secrets at once.
 
 ### Secrets in manual installations
 
-In the [manual setup](../platform-deployment/manual.md#run-connectors), inject secrets during Connector execution by providing
+In the [manual setup](../platform-deployment/local.md#run-connectors), inject secrets during Connector execution by providing
```
Comment (tmetzke) [inline versioned_docs/version-8.1/self-managed/connectors-deployment/connectors-configuration.md:105]:
```suggestion
In the [manual setup](../platform-deployment/manual.md#run-connectors), inject secrets during Connector execution by providing
```

**Comment by tmetzke** [comment]
> @christinaausley there is a build issue on your commit. Can you take a look?

Fixed it, I hope

**Comment by tmetzke** [comment]
Looks good now, needs another approval 🙂

**Review by christinaausley — APPROVED** [review]

**Pull request merged**